presidio-analyzer/analyzer/recognizer_result.py

```diff
@@ -13,3 +14,9 @@ def __init__(self, entity_type, start, end, score):
         self.start = start
         self.end = end
         self.score = score
+        self.interpretability_details = {}
```

Comment:
maybe 'explanations' or 'result_description' is a better phrase than 'interpretability_details'. Interpretability probably means more to us than to the new user

Reply:
👍